setup.py

```diff
@@ -104,7 +104,12 @@ def run(self):
         "yarl",
     ],
     extras_require={
-        "examples": ["aiomonitor", "click", "click-log", "click-completion"]
+        "examples": ["aiomonitor", "click", "click-log", "click-completion"],
+        "docs": [
+            "sphinx~=3.0",
+            "sphinx_rtd_theme~=0.4.3",
+            "sphinx_autodoc_typehints~=1.10",
```

Comment:
I also recommend `sphinxcontrib-trio`. It fixes some odd `async` related issues in sphinx